swarm/api/services/spec_manager.py

```
@@ -503,27 +503,31 @@ def list_runs(self, limit: int = 20) -> List[Dict[str, Any]]:
+                if state_file.exists():
+                    try:
+                        state = json.loads(state_file.read_text(encoding="utf-8"))
+                        runs.append(
+                            {
+                                "run_id": state.get("run_id", entry.name),
+                                "flow_key": state.get("flow_key"),
+                                "status": state.get("status"),
+                                "timestamp": state.get("timestamp"),
+                            }
+                        )
+                    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)